lego/apps/events/serializers/registrations.py

```diff
@@ -49,8 +49,8 @@ class RegistrationCreateAndUpdateSerializer(BasisModelSerializer):
         required=False, choices=constants.PAYMENT_STATUS_CHOICES
     )
     presence = PresenceField(required=False, choices=constants.PRESENCE_CHOICES)
-    photo_consent = ConsentField(
-        required=False, choices=constants.PHOTO_CONSENT_CHOICES
+    LEGACY_photo_consent = ConsentField(
+        required=False, choices=constants.LEGACY_PHOTO_CONSENT_CHOICES
```

Comment:
You can remove this, we don't need (or want) users to create or update legacy consents :grimacing:

Reply:
true. Removed